projects/demo/src/modules/components/dialog/dialog.template.html

```diff
@@ -90,6 +90,15 @@
         >
             <tui-dialog-example-5></tui-dialog-example-5>
         </tui-doc-example>
+
+        <tui-doc-example
+            id="mobile"
```

Comment:
this page already has element with `id="mobile"`.
Let's take another id.

Reply:
fixed